### PR TITLE
Removed translation for "etag" and changed spelling

### PR DESCRIPTION
**Pull request by rakekniven**
Shouldn't be translated.

Just checked https://en.wikipedia.org/wiki/HTTP_ETag and all available localized pages.

Signed-off-by: rakekniven <mark.ziegler@rakekniven.de>

### CODE REVIEW

File: src/libsync/discovery.cpp [hunk 1]
```diff
@@ -409,7 +409,7 @@ void ProcessDirectoryJob::processFileAnalyzeRemoteInfo(
         if (serverEntry.remotePerm.isNull())
             missingData.append(tr("permissions"));
         if (serverEntry.etag.isEmpty())
-            missingData.append(tr("etag"));
+            missingData.append("ETag");
```
Comment (er-vin) [inline src/libsync/discovery.cpp:412]: Should be `QStringLiteral("ETag")`. I didn't look the exact error yet but I guess that's why the CI is failing this. ;-)

**Review by er-vin — CHANGES_REQUESTED** [review]

File: src/libsync/discovery.cpp (hunk 1 above)
Comment (er-vin) [reply src/libsync/discovery.cpp:412]: And I was wrong, the CI didn't barf on this (although I still advise this should be done).

**Comment by er-vin** [comment]
Some tests need to be adjusted since they expect that translated etag string it seems.

**Comment by rakekniven** [comment]
Sorry, my coding skillz are not sufficient :-(

**Comment by er-vin** [comment]
> Sorry, my coding skillz are not sufficient :-(

Probably just need to change line 170 of src/test/testremotediscovery.cpp
It still carries "etag" you likely want to change it with "ETag"

File: src/libsync/discovery.cpp (hunk 1 above)
Comment (rakekniven) [reply src/libsync/discovery.cpp:412]: Changes are done and tests are ok.

**Comment by rakekniven** [comment]
> > Sorry, my coding skillz are not sufficient :-(
> 
> Probably just need to change line 170 of src/test/testremotediscovery.cpp
> It still carries "etag" you likely want to change it with "ETag"

Done. Thanks for pointing me to it.

**Review by er-vin — APPROVED** [review]

**Pull request merged**